### DISCUSSION TITLE
Marker genes

### DISCUSSION THREAD

**Discussion by mdhasan8** [BayraktarLab/cell2location#174] [General]
Hi,

I tried to run the cell2location but I didn't find the marker genes for figure 5(e). What are the marker genes for the human lymph node cell type T helper Tfh GC mapping? Would you please share the marker genes for these cell types?  or the dataset from which I can obtain the marker genes for human lymph node cell types? 

Thank you so much!
Best,
Easin

**Comment by vitkl** [comment]
Hi @mdhasan8 

I guess that you mean reference cell-type signatures of the immune cell types used in Fig 5. The reference dataset used was published by Elmentaite et al and can be downloaded as mentioned in the corresponding paper. You can then train the cell2location regression model to get cell-type signatures.

**Reply by mdhasan8** [reply]
Hello @vitkl,

Thank you so much for your reply. 
Actually, I am looking for the list of the marker genes for the cell types of Human Gut, Human Lymph Node, and Mouse brain Visium datasets. Do you have the marker genes files for the cell types in Figures 4(f) and 5d(Tfh, and CD8 Tmem)? I am trying to figure out the effects of marker genes for individual cell types in the cell2location  model. Please let me know.

**Comment by mdhasan8** [comment]
Hello @vitkl,

I have downloaded the covariate_effects.csv file from the following link: https://github.com/vitkl/cell2location_paper/tree/master/notebooks/selected_results/lymph_nodes_analysis/regression_model/RegressionNBV4Torch_57covariates_73260cells_10237genes

Do you think the values represent the expression signatures of reference cell types g_f,g or the expression of the level of genes mu_s,g (dependent variable)? or do these values represent the weights w_s,f  of each reference signature? 
Would you please let me know?
